src/Context/Type.hs

```diff
@@ -0,0 +1,20 @@
+module Context.Type where
+
+import Hadrian.Package.Type
+import Stage.Type
+import Way
```

Comment:
Presumably, this can be relaxed to `import Way.Type`?

Reply:
Addressed in an upcoming commit.